### PR TITLE
CIE-4217.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/configuration-bundle/configuring-security.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The Edge appliance is security hardened by default.
 Furthermore, you can apply specific security configurations to enhance the security of your Edge appliance.
 
 {{< c8y-admon-important >}}
-For information about deployment security best practices and tenant administration security configuration guidelines, see <a href="/files/edge/10-13-0_Security_Hardening_Guidelines_guide.pdf" target="_blank">Security Hardening Guidelines</a>.
+For information about deployment security best practices and tenant administration security configuration guidelines, see <a href="/files/edge/10-15-0_Security_Hardening_Guidelines_guide.pdf" target="_blank">Security Hardening Guidelines</a>.
```
Comment (BeateRixen) [inline content/edge/configuration-bundle/configuring-security.md:11]: @sharath-jk @sarangovindarajan  I was not aware that we share this document publicly. Usually we only provide access to the documents targeting at platform administrators via Empower. Do we have to split this document into an official part for tenant admins and an unofficial part for platform admins?

Comment (sarangovindarajan) [reply content/edge/configuration-bundle/configuring-security.md:11]: @BeateRixen It is currently structured to provide security best practices aimed for Tenant administrators and Deployment best practices aimed for Operations team. I am open to align this document with rest of the documentation practices followed. Let us discuss about split.

Comment (rssag) [reply content/edge/configuration-bundle/configuring-security.md:11]: i'd also advise against hosting pdf files in github

Comment (BeateRixen) [reply content/edge/configuration-bundle/configuring-security.md:11]: As agreed with @sarangovindarajan , we might split this document in a future release. For now, I recommend to not include the PDF here but link to Empower even if this is not accessible for everyone.

Comment (BeateRixen) [reply content/edge/configuration-bundle/configuring-security.md:11]: As just discussed, we will include the PDF file until the 10.15 operations documentation is available on Empower and then @sharath-jk will replace it with a link.

Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-security.md]: As agreed with Beate, we will have the PDF in the repo until the 10.15 Security Hardening PDF is published on Empower. After that, we will remove the PDF from GitHub repo and replace it with the Empower link to the PDF.

**Review by bbyreddy — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**